Header header
float32 angle
float32 bin_distance_step
provider_sonar/IntensityBin[] bins
